app/helpers/application_helper.rb

```diff
@@ -1,2 +1,23 @@
 module ApplicationHelper
 end
+
+class ActionView::Helpers::FormBuilder
```

Comment:
That sounds appropriate for the current state of it. If anyone extends it further, they should rename the file imho.